cpp/ql/src/semmle/code/cpp/models/implementations/Memset.qll

```diff
@@ -0,0 +1,38 @@
+import semmle.code.cpp.Function
+import semmle.code.cpp.models.interfaces.ArrayFunction
+import semmle.code.cpp.models.interfaces.DataFlow
+import semmle.code.cpp.models.interfaces.Taint
+
+/**
+ * The standard function `memset` and its assorted variants
+ */
+class MemsetFunction extends ArrayFunction, DataFlowFunction, TaintFunction {
+  MemsetFunction() {
+    this instanceof TopLevelFunction and
+    (
+      this.hasName("memset") or
+      this.hasName("__builtin_memset") or
+      this.hasName("FillMemory")
```

Comment:
It looks like [`FillMemory` is a macro](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/legacy/aa366561(v%3Dvs.85)).

Apparently `ZeroMemory` and `RtlZeroMemory` are macros too, but `bzero` (used on BSD and Mac) is a real function.

Reply:
Good catch.  They both resolve to... `memset`.